mod/quiz/backup/moodle2/restore_quiz_stepslib.php

```diff
@@ -292,6 +297,50 @@ protected function process_quiz($data) {
         }
     }
 
+    protected function process_quiz_question_legacy_instance($data) {
```

Comment:
I'm not sure why other processes don't have function docs, except for `process_quiz_slot_tags` I'd say we can add it to avoid ci issue.